client/src/components/Follow.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { ReactElement, useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { UserProfile } from '../../../types';
+import axios from 'axios';
+
+const Follow = (): ReactElement => {
+  const { id } = useParams<{ id: string }>();
+  const [user, setUser] = useState<UserProfile | null>(null);
+  const [isFollowing, setIsFollowing] = useState(false);
+
+  const follow = () => {
+    axios.post(`/api/follows/follow/${user?.id}/${id}`).then(() => {
```

Comment:
You can reduce this endpoint by removing the user.id param and get it from the req.user